test/unit/trainers/uri_trainer_test.rb

```diff
@@ -85,16 +85,16 @@ def test_save_selected_presets_overrides_the_config
       end
 
       def test_save_selected_presets_saves_overwrite
-        domain = NewsScraper::Transformers::Article.scrape_patterns['domains'].keys.first
+        domain = NewsScraper::Constants::SCRAPE_PATTERNS['domains'].keys.first
```

Comment:
Since you wrapped this in module NewsScraper, I don't think you'll need the prefix here (and below)